docs/content/components/forms.md

```diff
@@ -289,7 +317,18 @@ If the input should show a **warning**, add the `.warn` class to the `<div class
 </form>
 ```
 
-#### Checkboxes and radios
+### Form actions
+
+Align buttons to the right—via `float: right` on the buttons—in forms with `.form-actions`. The floats are automatically cleared for you.
+
+```html live
+<div class="form-actions">
+  <button type="submit" class="btn btn-primary">Save changes</button>
+  <button type="button" class="btn">Cancel</button>
+</div>
+```
+
+## Checkboxes and radios
 
 Utilities to spice up the alignment and styling of checkbox and radio controls.
```

Comment:
I know it's not the point of the PR, but if you can think of another way to say this instead of "spice up". I understand the meaning of it, but don't actually know what these utilities do based on that term, and it's hard to tell based on the example 🙂